website/source/guides/operating-a-job/advanced-scheduling/preemption-service-batch.html.md

```diff
@@ -0,0 +1,447 @@
+---
+layout: "guides"
+page_title: "Preemption (Service and Batch Jobs)"
+sidebar_current: "guides-operating-a-job-preemption-service-batch"
+description: |-
+  The following guide walks the user through enabling and using preemption on
+  service and batch jobs in Nomad Enterprise (0.9.3 and above). 
+---
+
+# Preemption for Service and Batch Jobs
+
+~> **Enterprise Only!** This functionality only exists in Nomad Enterprise. This
+is not present in the open source version of Nomad.
+
+Prior to Nomad 0.9, job [priority][priority] in Nomad was used to process
+scheduling requests in priority order. Preemption, implemented in Nomad 0.9
+allows Nomad to evict running allocations to place jobs of a higher priority.
+Jobs whose allocations are blocked temporarily go into "pending" status until
+the cluster has additional capacity to run it. This allows operators to run high
+priority jobs even under resource contention across the cluster.
+
+While Nomad 0.9 introduced preemption for [system][system-job] jobs, Nomad 0.9.3
+[Enterprise][enterprise] additionally allows preemption for
+[service][service-job] and [batch][batch-job] jobs.
+
+## Reference Material
+
+- [Preemption][preemption]
+- [Nomad Enterprise Preemption][enterprise-preemption]
+
+## Estimated Time to Complete
+
+20 minutes
+
```

Comment:
Would suggest removing the Challenge/solution format and instead replacing with sections like:

When is Preemption useful? 

Enabling/Disabling Preemption

Preemption Example: 
 - Preerequistes
 - Create low priority job
 - Create high priority job
 - Enable preemption
 - Nomad plan shows preemptions
 - Nomad run shows eviction of lower priority job. 

Conclusion

Reply:
Removed challenge/solution and re-worded intro to add more context including why preemption would be useful and how to enable/disable it.